pkg/local_object_storage/shard/mode.go

```diff
@@ -1,50 +1,43 @@
 package shard
 
+import "errors"
+
 // Mode represents enumeration of Shard work modes.
 type Mode uint32
 
+// ErrReadOnlyMode is returned when it is impossible to apply operation
+// that changes shard's memory due to the "read-only" shard's mode.
+var ErrReadOnlyMode = errors.New("shard is in read-only mode")
+
 // TODO: more detailed description of shard modes.
 
 const (
 	_ Mode = iota
 
-	// ModeActive is a Mode value for active shard.
-	ModeActive
-
-	// ModeInactive is a Mode value for inactive shard.
-	ModeInactive
+	// ModeReadWrite is a Mode value for shard that is available
+	// for read and write operations. Default shard mode.
+	ModeReadWrite
```

Comment:
May be it's time to have `ReadWrite` as default mode (0)?

Reply:
have nothing against it

made it default